docs/source/development/set_up_pycharm.md

```diff
@@ -50,22 +50,16 @@ Add a new **Python** Run configuration:
 
 ![](../meta/images/pycharm_add_py_run_config.png)
 
-Create a **Run / Debug Configuration** for `kedro run` and get the path to the Kedro CLI script:
+Specify the **Run / Debug Configuration** name in the **Name** field, and edit the new Run configuration as follows:
 
-```console
-# macOS / Linux
-which kedro
-
-# Windows (in `cmd`)
-python -c "import sys, os.path; print(os.path.join(os.path.dirname(sys.executable), 'Lib\site-packages\kedro\__main__.py'))"
-```
-
-Edit the new Run configuration as follows:
+- Pick **Module** from the dropdown
+- Enter ``kedro`` in the **Module Name** field
+- Enter ``run`` in the **Parameters** field
+- Enter the path of your project directory into the **Working directory** field
+- Pick ``Emulate termulate in output consule`` from the **Modify options** dropdown, and then click **OK**
```

Comment:
```suggestion
- Pick ``Emulate terminal in output consule`` from the **Modify options** dropdown, and then click **OK**
```

Reply:
"Termulate" is a new typo for me! :D

Reply:
I think you want "console" as well, right?